src/styles/getOptionRowStyles/index.js

```diff
@@ -0,0 +1,10 @@
+import styles from '../styles';
```

Comment:
On second thought, this doesn't have to be a function, so the folder should just be called `optionRowStyles`

Reply:
Fixed.